static/css/navbar.css

```diff
@@ -30,6 +30,9 @@
 .navbar a:hover, .subnav-right:hover .subnavbtn {
   background-color: black;
 }
+.navbar a img {
+  height: 20px;
```

Comment:
still needed?

Reply:
not if we get rid of the image and just have the version in a red bubble instead of the 'X' image when there are errors